src/silkaj.py

```diff
@@ -1,51 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ok, I put it on next line [like here](https://github.com/YunoHost/package_linter/blob/master/package_linter.py#L1-L2) it works.
I am calling Silkaj with `./silkaj.py` which go searching first line executor (python3).